components/buttonform/buttonform-solid.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../lib/units";
+
+
+@mixin buttonform-solid(){
+    &.mmf-form--button {
+        
+        .mmf-label {
+            display: block;
+            padding-bottom: $mmf-unit;
+            @include solid-font-style();
```

Comment:
by convention, place the include as first statement of css-rules. This ensures, your display and padding are not overwritten by the include